src/style.css

```
@@ -1,3 +1,73 @@
+:root {
+  --main-color: rgb(0, 0, 0);
+  --color-links: rgb(0, 172, 220);
+  --bg-color-primary: rgb(255, 255, 255);
+  --shadow-color: rgba(0, 0, 0, 0.05);
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 var(--shadow-color);
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
   margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  position: relative;
+  line-height: 60px;
+  color: var(--main-color);
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: var(--color-links);
+}
+
+.nav__link.is-active {
+  color: var(--color-links);
+}
+
+.nav__link.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: var(--color-links);
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

u can improve ur code
```suggestion
.nav_item:not(:first-child) {
  margin-left: 20px;
}
```